### DISCUSSION TITLE
I would more promote that the app works only offline and set it as one of rules for future issues, PRs etc.

### DISCUSSION THREAD

**Discussion by inson1** [clementwzk/OpenCalc#344] [Ideas]
What do you think?

**Comment by clementwzk** [comment]
Yes we should write the contributing guidlines

**Reply by inson1** [reply]
@Darkempire78 And maybe add it to app info "A simple and beautiful calculator for Android" ?

Lets say "A simple, offline and beautiful calculator for Android" ? 
Or "A simple and beautiful offline calculator for Android" ?

**Reply by inson1** [reply]
@Darkempire78 What do you think about that?
